doc/api/esm.md

```diff
@@ -1362,9 +1374,7 @@ and there is no security.
 // https-loader.mjs
 import { get } from 'https';
 
-export function resolve(specifier, context, defaultResolve) {
-  const { parentURL = null } = context;
-
+export function resolve(specifier, parentURL) {
```

Comment:
Is this a mistake? Shouldn't this be `export function resolve(specifier, context, parentLoader)`?